<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Elpida::Application::MemoryOverheadView</class>
 <widget class="QWidget" name="MemoryOverheadView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="gbMemoryOverheadResults">
     <property name="title">
      <string>Results</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeWidget" name="twMemoryOverheadResults">
        <column>
         <property name="text">
          <string>Wokring set size</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Scale</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Sub Samples</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Expected score</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Actual score</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Overhead</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="gbExecution">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Execution</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
        <item>
         <widget class="QLabel" name="lblCurrentBenchmark">
          <property name="text">
           <string>Current benchmark:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblCurrentBenchmarkValue">
          <property name="minimumSize">
           <size>
            <width>256</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QProgressBar" name="pbProgress"/>
        </item>
        <item>
         <widget class="QLabel" name="lblProgress">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>0/0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="bpStart">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblStatus">
          <property name="text">
           <string>Ready</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="gbBenchmarkConfiguration">
     <property name="title">
      <string>Benchmark Configuration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="spnSubSamplesMultiplier">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>0.001000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="spnInitialScale">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>0.000100000000000</double>
        </property>
        <property name="value">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblMultiplier">
        <property name="text">
         <string>Scale multiplier:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblInitialScale">
        <property name="text">
         <string>Start scale:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblInitialSubSamples">
        <property name="text">
         <string>Start subsamples:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblSubsamplesMultiplier">
        <property name="text">
         <string>Subsamples multiplier:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spnInitialSubSamples">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000000000</number>
        </property>
        <property name="value">
         <number>32</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="spnScaleMultiplier">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblIterations">
        <property name="text">
         <string>Iterations:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="spnIterations">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>8</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
